internal/pkg/client/library/library.go

```diff
@@ -74,11 +74,15 @@ func getDownloadConfig() (libClient.Downloader, error) {
 	// get downloader parameters from config
 	conf := apptainerconf.GetCurrentConfig()
 	if conf == nil {
-		// sylog.Fatalf("Unable to get apptainer configuration")
-		var err error
-		conf, err = apptainerconf.Parse(buildcfg.APPTAINER_CONF_FILE)
-		if err != nil {
-			sylog.Fatalf("unable to parse apptainer.conf file: %s", err)
+		if strings.HasSuffix(os.Args[0], ".test") {
```

Comment:
Instead of having this directly in the code, I would put the parsing and initialization of current config directly in `_test.go` files

Reply:
Likewise I found that this one was also called from other `_test.go` files in addition to its companion, so this also needs to stay.  The previous code was already loading it here; my change just makes sure that it is only applied during unit tests.